tests/test_starfile_stack.py

```diff
@@ -30,6 +32,9 @@ def run(self, result=None):
                 should_delete_folder = True
 
             temp_file_path = os.path.join(temp_folder_path, "sample.mrcs")
```

Comment:
I know its not a line that you touched but while we are in here we might be able to improve the tempfolder path.  Should be able to find better examples (using builtin functions/contexts) elsewhere in the unit tests.